webtasks/cron_mongodb.js

```diff
@@ -0,0 +1,230 @@
+var _ = require('lodash');
+var Bluebird = require('bluebird');
+
+var mongo;
+
+
+var webtask = function (context, req, res) {
+    var action = validate_method({
+        put_job: 'PUT',
+        get_job: 'GET',
+        destroy_job: 'DELETE',
+        list_jobs: 'GET',
+        job_history: 'GET',
+        reserve_jobs: 'POST',
+        renew_reservation: 'PUT',
+        release_reservation: 'DELETE',
+    });
+    
+    if (!validate_params(['MONGO_COLLECTION']))
+        return;
+    
+    if (action === 'put_job') {
+        // Create or update a scheduled webtask
+        
+        if (!validate_params(['container', 'name', 'schedule', 'token']))
+            return;
+            
+        var now = new Date();
+        
+        var item = {
+            state: 'active',
+            schedule: context.data.schedule,
+            token: context.data.token,
+            container: context.data.container,
+            name: context.data.name,
+            last_run_at: null,
+            next_available_at: now,
```

Comment:
As discussed on Slack, the "now" may not necessarily be true based on schedule, but maybe we can have the worker figure it out to avoid code duplication with proxy.